projects/Odroid_C2/linux/linux.aarch64.conf

```diff
@@ -3361,15 +3402,16 @@ CONFIG_HFSPLUS_FS=y
 # CONFIG_BFS_FS is not set
 # CONFIG_EFS_FS is not set
 # CONFIG_JFFS2_FS is not set
+# CONFIG_UBIFS_FS is not set
 # CONFIG_LOGFS is not set
 # CONFIG_CRAMFS is not set
 CONFIG_SQUASHFS=y
-CONFIG_SQUASHFS_FILE_CACHE=y
-# CONFIG_SQUASHFS_FILE_DIRECT is not set
-CONFIG_SQUASHFS_DECOMP_SINGLE=y
+# CONFIG_SQUASHFS_FILE_CACHE is not set
```

Comment:
@stefansaraev Why do you prefer FILE_CACHE over FILE_DIRECT? I'm curious if you've seen performance benefits from the former?